test/unit/web_page_media_card.spec.js

```diff
@@ -106,7 +106,7 @@ describe('<WebPageMediaCard />', () => {
       dbid: 10,
       quote: null,
       published: '1506728130',
-      archived: false,
+      archived: 0,
```

Comment:
Is there a reason for not importing `CheckArchivedFlags` and using the constants from the tests too?

Reply:
No as it's a test I used the value direct, will replace it with the constants

Reply:
Fixed by using CheckArchivedFlags constants